mockgcp/README.md

```diff
@@ -97,4 +99,18 @@ We will compare new HTTP logs with the golden HTTP log as a part of the mockGCP
 
 To prevent any problems when comparing with the golden logs, it is necessary to replace all generated values in the HTTP log with identical values.
 
-See [here](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/unified_test.go#L167-L329) to get some ideas.
\ No newline at end of file
+See [here](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/unified_test.go#L167-L329) to get some ideas.
+
+## Appendix
+
+### Generating proto
+
+If the proto file on [googleapis](https://github.com/googleapis/googleapis/commits/1e4137870560340a14700618a05e2d7162326af7/google/cloud/ids/v1/ids.proto) is out of date or non existent, you can generate a proto file from the [google generated api](https://github.com/googleapis/google-api-go-client/tree/b49e3b908a8ed562e068736f1c42e992538ba6e0) as such:
```

Comment:
Nit: Maybe make it clearer that this should be added to the `generate-openapis` Makefile target?

Reply:
Spelled it out below